intro-HPC/antwerpen/macros.tex

```diff
@@ -7,6 +7,8 @@
 \macro{\hpcname}{hopper}
 \macro{\loginnode}{login.hpc.uantwerpen.be}
 \macro{\loginhost}{ln02.hopper.uantwerpen.vsc}
+%TODO: RSA key fingerprint here
```

Comment:
the line below it should contain the correct fingerprint, I didn't actually check it, just copied the one that was currently showing up, so this is not a regression, but could be enhanced.